### PR TITLE
Implement tests

### PR DESCRIPTION
**Pull request by csadorf**

### CODE REVIEW

**Comment by csadorf** [comment]
I've discovered an issue with these tests, specifically the test are executed for `apps/home`, but the version to be tested is mounted as `apps/app`.

File: tests/test_start.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    handles = set()
+    wh_before = selenium.window_handles
+    yield handles
+    time.sleep(timeout)
+    handles.update(set(selenium.window_handles).difference(wh_before))
+
+
+def test_click_appstore(selenium, url):
+    selenium.get(url("apps/apps/home/start.ipynb"))
+    with get_new_windows(selenium) as handles:
```
Comment (yakutovicha) [inline tests/test_start.py:17]:
```suggestion
    with get_new_windows(selenium, timeout=5) as handles:
```

I am not sure if 5 seconds is long enough. However, I am sure that 2 seconds are too short, as we don't yet see the apps.

**Review by yakutovicha — COMMENTED** [review]
Thanks, @csadorf, looks awesome.

I have one little suggestion and one question.

q: can we also test the terminal?

File: tests/test_start.py (hunk 1 above)
Comment (csadorf) [reply tests/test_start.py:17]: This timeout is only concerned for how long to wait for new windows. If we need more time for certain elements to appear (e.g. apps on the app store), then we should either query for a specific element or add an additional sleep delay.

**Comment by csadorf** [comment]
> Thanks, @csadorf, looks awesome.

Thank's. 😃 

> q: can we also test the terminal?

Yes, I'll add a test for that.

File: tests/test_start.py (hunk 1 above)
Comment (yakutovicha) [reply tests/test_start.py:17]: Ok. Can you please put the sleep delay then?

**Review by dou-du — APPROVED** [review]

**Comment by csadorf** [comment]
@dou-du Would you be able to make those last requested changes?

**Comment by dou-du** [comment]
> @dou-du Would you be able to make those last requested changes?

Sorry, I am not very clear. Do you want me to implement the test for the terminal ?

**Comment by csadorf** [comment]
> > @dou-du Would you be able to make those last requested changes?
> 
> Sorry, I am not very clear. Do you want me to implement the sleepy delay and the test for the terminal ? By the way, I do not
> have permission to edit for this repository.

Yes, exactly. That would be a good chance to get familiar with how to implement these tests. We can give you access, but since this PR is from my fork, you would want to fork this repository and then create the PR into my branch anyways.

**Comment by dou-du** [comment]
> > > @dou-du Would you be able to make those last requested changes?
> > 
> > 
> > Sorry, I am not very clear. Do you want me to implement the sleepy delay and the test for the terminal ? By the way, I do not
> > have permission to edit for this repository.
> 
> Yes, exactly. That would be a good chance to get familiar with how to implement these tests. We can give you access, but since this PR is from my fork, you would want to fork this repository and then create the PR into my branch anyways.

Sure, I will do it. Thanks.

**Review by yakutovicha — APPROVED** [review]
@dou-du please make the discussed changes in a separate PR. I will approve/merge that one.

**Pull request merged**